.github/workflows/labeler.yml

```
@@ -1,15 +1,31 @@
-
-jobs:
-  label:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/labeler@v5
-        with:
-          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+# Label families → keep to the canonical names from labels-guide
+
+# Head branch → status hints (works on PRs)
+"status:needs-review":
```

actionlint (1.7.7)

unexpected key "status:needs-review" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)